platform/viewer/src/lib/DICOMSR/retrieveDataFromSR.js

```diff
@@ -0,0 +1,70 @@
+import * as dcmjs from 'dcmjs';
+import { getAllDisplaySets, getInstanceMetadata } from './srUtils';
```

Comment:
Are these exposed outside of the DICOM SR logic? (are they public for consumers of `@ohif/core`?)

Reply:
Not right now.